frontend/src/config/globals.jsx

```diff
@@ -1,26 +1,5 @@
-// Anything that was defined in /public/env-config.js (this file is created via docker for deployment reasons so we can
-// specify the API_URL, for example)
-let windowEnvironment = window._env_;
-if (windowEnvironment === undefined) {
-  windowEnvironment = {};
-}
-
 export const Constants = {
-  api:
-    windowEnvironment.REACT_APP_API_URL ||
-    process.env.REACT_APP_API_URL ||
-    'https://code4rocoviz19api-demo.azurewebsites.net/api/v2',
-  isDev:
-    windowEnvironment.ENV === 'development' ||
-    process.env.NODE_ENV === 'development',
-  isProd:
-    windowEnvironment.ENV === 'production' ||
-    process.env.NODE_ENV === 'production',
-
-  specifyUnknownData:
-    (windowEnvironment.SPECIFY_UNKNOWN_DATA ||
-      process.env.REACT_APP_SPECIFY_UNKNOWN_DATA ||
-      'false') === 'true',
+  api: process.env.REACT_APP_API_URL,
```

Comment:
how will we pass the API url? at build time?

Reply:
Yes, take a look here https://github.com/code4romania/date-la-zi/pull/306/files#diff-46f99ccdfc1ed31dc64ca065888b9236R45

Reply:
Okay so we'll hard code them into the script

Reply:
FE not using the API anymore